core/controlplane/config/templates/cloud-config-controller

```diff
@@ -345,11 +347,16 @@ coreos:
       command: start
       content: |
         [Unit]
-        Wants=kubelet.service docker.service
-        After=kubelet.service
+        Wants=kubelet.service docker.service install-kube-system.service
+        After=kubelet.service install-kube-system.service
 
         [Service]
-        Type=oneshot
+        Type=simple
```

Comment:
I can't speak for the rest of the processes, but here simple was preferred because oneshot would exit if the check for install-kube-system failed at all. Changing to simple allowed the adding of the functionality of the status checking. Might be worth checking into changing others if it makes the whole sequence more robust, but I think it would be better suited for another PR.